README.md

```diff
@@ -15,7 +15,7 @@ Prerequisites:
 - Java SDK 1.8 (Build will not work on JDK 9 and above as of now)
 - Apache Maven
 
-Clone this repository and run `mvn install` in the root directory.
+Clone this repository, set the environment variable MVN\_HOME with the path to this repository's root directory and run `mvn install` in the root directory.
```

Comment:
This is a severe typo which swept over into the rest of the README. This should be NMF\_HOME since it has nothing to do with maven. I suggest the following update:
```suggestion
Clone this repository, set the environment variable NMF\_HOME with the path to this repository's root directory and run `mvn install` in the root directory.
```